KubeArmor/monitor/systemMonitor.go

```diff
@@ -169,7 +175,7 @@ func NewSystemMonitor(node *tp.Node, logger *fd.Feeder, containers *map[string]t
 
 	mon.Status = true
 	mon.UptimeTimeStamp = kl.GetUptimeTimestamp()
-	mon.HostByteOrder = bcc.GetHostByteOrder()
+	mon.HostByteOrder = binary.LittleEndian
```

Comment:
can we not hardcode the endianness?

Reply:
BCC used unsafe calls to determine the byte order
Ref https://github.com/iovisor/gobpf/blob/2289761f1e2092a7416cd5fd55a218802d997cb6/bcc/perf.go#L110

I would like to defer adding this till #614 and prefer using a better way like [conditional builds](https://groups.google.com/g/golang-nuts/c/3GEzwKfRRQw) to determine this. WDYT?

Reply:
sounds good. I believe ARM wont work anyways unless this gets fixed.